src/test/java/systemtests/ClearCommandSystemTest.java

```diff
@@ -65,7 +75,14 @@ public void clear() {
      * @see AddressBookSystemTest#assertApplicationDisplaysExpected(String, String, Model)
      */
     private void assertCommandSuccess(String command) {
-        assertCommandSuccess(command, ClearCommand.MESSAGE_SUCCESS, new ModelManager());
+        assertCommandSuccess(
+                command,
+                ClearCommand.MESSAGE_SUCCESS,
+                new ModelManager(
```

Comment:
Seeing this invocation quite a lot, maybe we can have a factory for storageStubModel possibly in TestUtil?

Reply:
Yup, created two more mocks for these.

Reply:
Actually, should we make `makeTempStorage` in TestUtil a mock as well?

Then we can do `new TempStorageManager()`

Reply:
Actually, makeTempStorage is completely unused. Removing.